content/docs/user-guide/tracking-existing-data.md

```diff
@@ -0,0 +1,90 @@
+# Tracking Existing Files and Directories
+
+You can use `dvc add` to track data located in your current
+<abbr>workspace</abbr>, or in the supported
+[external locations](/doc/user-guide/managing-external-data). See
+[Data Versioning](/doc/start/data-versioning) for a primer.
+
+Alternatively, `dvc import` and `dvc import-url` let you bring data from
+external locations to your project, and start tracking it at the same time. See
+also [Data Access](/doc/start/data-access).
+
+In any case, one or more files ending with the `.dvc` extension ("dot DVC file")
+are created in the project, containing the information to track the target data
+over time.
+
+> Note `.dvc` files are meant to be versioned with Git (if enabled in the
+> <abbr>repository</abbr>).
+
+## Dot DVC files
+
+`.dvc` files are placeholders that supports basic tracking of data files and
+directories in DVC. <abbr>DVC projects</abbr> typically have one `.dvc` file per
+data file or directory being tracked.
+
+These files use the [YAML 1.2](https://yaml.org/) file format, and a
+human-friendly schema described below. We encourage you to get familiar with it
+so you may modify, write, or generate `.dvc` files on your own. Here's an
+example:
+
+```yaml
+outs:
+  - md5: a304afb96060aad90176268345e10355
+    path: data.xml
+    desc: Cats and dogs dataset
+
+# Comments and user metadata are supported.
+meta:
+  name: 'Devee Bird'
+  email: devee@dvc.org
+```
+
+> See [How to Merge Conflicts](/doc/user-guide/how-to/merge-conflicts) for tips
+> on managing DVC files.
+
+## .dvc YAML specification
+
+| Field  | Description                                                                                                                                                                                                                   |
+| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `outs` | (Required) list of [output entries](#output-entries) (details below) that represent the files or directories tracked with DVC. Typically there is only one (but several can be added or combined manually).                   |
```

Comment:
Possible problem: while these lists [look ](https://dvc-landing-2-0-pipelin-rttezd.herokuapp.com/doc/user-guide/tracking-existing-data#dvc-yaml-specification)like a much better structured ref. they're painful to deal with in MD (our formatter makes all the lines of the same length).

Should we consider a specially designed element for these so they're easy to write/manage in md and yet look like well-structured tables? Example in https://nightly.spacy.io/usage/projects#project-yml